src/components/dnn-permissions-grid/dnn-permissions-grid.tsx

```diff
@@ -0,0 +1,580 @@
+import { Component, Host, h, Prop, State, Event, EventEmitter, Watch } from '@stencil/core';
+import { ILocalization } from './localisation-interface';
```

Comment:
```suggestion
import { ILocalization } from './localization-interface';
```

NOTE: This change will also require the filename to be changed.